### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by krishvoor**
This PR accommodates to requirement to add Oracle Instant Client for
ppc64le as well.

### CODE REVIEW

**Comment by krishvoor** [comment]
Will allow ppc64le specific [Oracle instant client](https://www.oracle.com/database/technologies/instant-client/linux-power-le-64-downloads.html) to be valued as well

**Comment by krishvoor** [comment]
Hi @eguzki @miguelsorianod  👋 ,
Please help with your review, updating Dockerfile so it can add instant client packages irrespective of architecture.

**Review by eguzki — DISMISSED** [review]

File: amp/system-oracle/Dockerfile [hunk 1]
```diff
@@ -2,9 +2,9 @@ FROM registry.redhat.io/3scale-amp2/system-rhel7:3scale2.11
 
 USER root
 
-COPY ./oracle-client-files/instantclient-basic*-linux.x64*.zip \
```
Comment (eguzki) [inline amp/system-oracle/Dockerfile:5]: `linux`  prefix will also work for `ppc64le`?

Comment (eguzki) [reply amp/system-oracle/Dockerfile:5]: How do ppc64le oracle client files look like?

Comment (krishvoor) [reply amp/system-oracle/Dockerfile:5]: The instantclient [package](https://download.oracle.com/otn_software/linux/instantclient/193/instantclient-basic-linux.leppc64.c64-19.3.0.0.0dbru.zip)  has *linux* prefix in it, so does SDK,odbc packages.

**Review by eguzki — APPROVED** [review]

**Review by eguzki — APPROVED** [review]

**Pull request merged**